### DISCUSSION TITLE
Swiping photos, first not sharp

### DISCUSSION THREAD

**Discussion by zwrmueller** [deckerst/aves#1283] [General]
Hi,
I observed that when I swipe through my photos in fullscreen for the very first moment the next photo is not sharp and becomes sharp the very next moment after.
I don't think it is a bug, but it annoys me a bit, a very very little bit ;-). 
Other gals I have don't show this. I did not find any parameter setting to stop this, so I think it might be inherent of the programming, may be flutter is a bit slow or whatsoever.

Anyonoe who can reproduce or has an idea about the reason of that behaviour?

**Comment by deckerst** [comment]
Related:
- #464

**Comment by zwrmueller** [comment]
Thanx @deckerst for this link. Then it is as I guessed and I can calm down and live with it.

Thanx for this nice program anyway!
